editor/src/utils/keyboard.ts

```diff
@@ -254,6 +254,13 @@ export const Keyboard = {
       return char === keyChar
     })
   },
+  keyIsArrow: function (keyChar: KeyCharacter): boolean {
+    return ['left', 'right', 'up', 'down'].some((char) => char === keyChar)
```

Comment:
Surely a switch statement makes more sense here?

Reply:
I just borrowed the pattern from the function above (`keyIsModifier`)
But surely I can make it a switch case.

Reply:
Fixed in 5ee2ed1f5201d1d7179d95afa51a3a95ecf4dea1